### PR TITLE
For discussion only: Changes to support scaife.perseus.org

### PR DESCRIPTION
**Pull request by jacobwegner**
@jtauber: before opening an upstream PR, I thought it might be useful to discuss here if the changes I've made should be made at the data level or if they should be made within how we parse nodes in XSLT.

Will update this thread with a preview URL

### CODE REVIEW

**Comment by jacobwegner** [comment]
Deployed to the dev instance via:

https://scaife-perseus-org-dev.herokuapp.com/reader/urn:cts:mayaLit:lit0001.pw0001.popolwuj-quc:1-5

**Comment by jtauber** [comment]
The trailing `:` in the URNs should be valid right? As in that's a limitation of Scaife, not a problem with the text.

For the rest of the changes, is the issue just our XSLT or actually how Capitains handles it. If the former, I'm inclined to support it in our XSLT but again I'll raise it with OGL but it's clearly _different_ from how the OGL texts are being done if nothing else.

**Comment by jacobwegner** [comment]
@jtauber:

> The trailing : in the URNs should be valid right? As in that's a limitation of Scaife, not a problem with the text.

It is actually a problem within MyCapytain (which won't handle the trailing `:` and also doesn't support exemplars):

**Comment by jacobwegner** [comment]
@jtauber:

> The trailing : in the URNs should be valid right? As in that's a limitation of Scaife, not a problem with the text.

It is actually a problem within MyCapytain (which won't handle the trailing `:` and also doesn't support exemplars):

https://github.com/Capitains/MyCapytain/blob/60d325e92ad41cdf3127f0c6008d5dc62093b063/MyCapytain/common/reference.py#L602

If we feel we want to patch the parsing within scaife-viewer/scaife-viewer and do a backport upstream, I can pursue that; otherwise changing the CTS data is the quickest path to deploy.

**Comment by jacobwegner** [comment]
@jtauber:

> For the rest of the changes, is the issue just our XSLT or actually how Capitains handles it. If the former, I'm inclined to support it in our XSLT but again I'll raise it with OGL but it's clearly different from how the OGL texts are being done if nothing else.

That's why I wanted your input; I believe we could work around the issues by changing our XSLT, but given that all of the other texts in the corpus are standardized, I thought it might be something to suggest to the curators.

(e.g. we'd need to special case some things in our XSLT for just these texts)

**Comment by jtauber** [comment]
Yep, I'll raise these issues on my Thursday call

**Comment by jacobwegner** [comment]
Needed to set `-spa` as a `translation` to get the language to show up:

image

https://scaife-dev.perseus.org/library/urn:cts:mayaLit:lit0001.pw0001/

and the rendering is fixed:

image

https://scaife-dev.perseus.org/reader/urn:cts:mayaLit:lit0001.pw0001.popolwuj-spa:1-5/

File: data/lit0001/pw0001/__cts__.xml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-  <ti:edition workUrn="urn:cts:mayaLit:lit0001.pw0001" urn="urn:cts:mayaLit:lit0001.pw0001.popolwuj-quc:">        
+  <ti:edition workUrn="urn:cts:mayaLit:lit0001.pw0001" urn="urn:cts:mayaLit:lit0001.pw0001.popolwuj-quc">        
     <ti:label xml:lang="quc">Popol Vuh</ti:label>
     <ti:description xml:lang="eng">Ximenez copy, K'iche' column</ti:description>
    </ti:edition>         
   
-    <ti:edition workUrn="urn:cts:mayaLit:lit0001.pw0001" urn="urn:cts:mayaLit:lit0001.pw0001.popolwuj-spa:">        
-      <ti:label xml:lang="quc">Popol Vuh</ti:label>
+    <ti:translation workUrn="urn:cts:mayaLit:lit0001.pw0001" urn="urn:cts:mayaLit:lit0001.pw0001.popolwuj-spa" xml:lang="spa">        
+      <ti:label xml:lang="spa">Popol Vuh</ti:label>
```
Comment (jtauber) [inline data/lit0001/pw0001/__cts__.xml]: This should be the language of the label itself so `quc` is correct, I think

Comment (jacobwegner) [reply data/lit0001/pw0001/__cts__.xml]: @jtauber good catch; was left over from my attempts to resolve metadata issues.  Have reverted via 4030c79.

**Comment by jacobwegner** [comment]
@jtauber I opened https://github.com/scaife-viewer/scaife-viewer/issues/445 to track adding a `ti:text` kind to scaife.perseus.org; it'd require a lot of changes to our MyCapytain resolver, so I'm going to move forward with calling both of the versions here "translations".

**Comment by jacobwegner** [comment]
I've also opened https://github.com/scaife-viewer/scaife-viewer/issues/446 to handle the trailing colon issue

**Comment by jacobwegner** [comment]
Closing to re-open against _upstream_.

**Pull request closed without merging**